File: feature-map/src/main/java/com/levnovikov/feature_map/dependency/MapProvider.java
package com.levnovikov.feature_map.dependency;

import com.google.android.gms.maps.GoogleMap;

import io.reactivex.Maybe;

/**
 * Author: lev.novikov
 * Date: 27/1/18.
 */

public interface MapProvider {
    Maybe<GoogleMap> getMap();
}

File: feature-map/src/main/java/com/levnovikov/feature_map/dependency/MapProvider.kt
package com.levnovikov.feature_map.dependency

import com.google.android.gms.maps.GoogleMap

import io.reactivex.Maybe

/**
 * Author: lev.novikov
 * Date: 27/1/18.
 */

interface MapProvider {
    val map: Maybe<GoogleMap>
}
